packages/frontend-2/components/settings/shared/ChangeRoleDialog.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <LayoutDialog v-model:open="isOpen" max-width="sm" :buttons="dialogButtons">
+    <template #header>Change role</template>
+    <div class="flex flex-col gap-4 text-body-xs text-foreground">
+      <p>Are you sure you want to change the role of the selected user?</p>
+      <div v-if="newRole && oldRole" class="flex flex-col gap-3">
+        <div class="flex items-center gap-2 font-semibold">
+          {{ name }}
+        </div>
+        <div class="flex gap-2 items-center">
+          <span>{{ getRoleLabel(oldRole).title }}</span>
+          <ArrowRightIcon class="h-4 w-4" />
+          <span>{{ getRoleLabel(newRole).title }}</span>
+        </div>
+      </div>
+    </div>
+  </LayoutDialog>
+</template>
+
+<script setup lang="ts">
+import { LayoutDialog, type LayoutDialogButton } from '@speckle/ui-components'
```

Comment:
You don't need to import components from ui-components, they're auto-imported with the help of @speckle/ui-components-nuxt